package.json

```diff
@@ -116,7 +115,9 @@
     "ms": "^2.1.2",
     "ncp": "^2.0.0",
     "node-fetch": "^2.6.0",
+    "node-forge": "^1.3.1",
```

Comment:
Is this needed? It is not referenced from js and I don't see any reference to it in `selfsigned` docs.

Reply:
Selfsigned was updated recently and these missing dependecies are now included to package.